test/helpers/utils.js

```diff
@@ -13,24 +13,24 @@ window.Utils = {
         if (opts.label) {
             var $text = $el.find('.guide_label').eq(idx);
             expect($text.text()).toBe(opts.label.text);
-            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, 0);
-            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, 0);
+            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, -1);
+            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, -1);
```

Comment:
Ah ok, that's very unintuitive (don't tell Shannon I used that word!), but thanks for the explanation. It clearly gets the job done :+1: